geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala

```diff
@@ -89,8 +92,31 @@ object AccumuloDataStoreFactory {
     val instance = instanceIdParam.lookup[String](params)
     val user = userParam.lookup[String](params)
     val password = passwordParam.lookup[String](params)
+    val keytabPath = keytabPathParam.lookup[String](params)
+
+    var usingKerberos = false
+
+    // Build authenticaton token according to how we are authenticating
+    val authToken : AuthenticationToken = if (password != null ) {
+      new PasswordToken(password.getBytes("UTF-8"))
+    } else if (keytabPath != null && !useMock) {
+      usingKerberos = true
```

Comment:
It looks like the only situation where 'usingKerberos' is true is when "keytabPath != null && !useMock" is true.  

It might be worth it to ditch the var, and then check that condition below.